enkelt.py

```diff
@@ -95,6 +99,26 @@ def e():
         def pi():
             return math.pi
 
+    class tid:
+        @staticmethod
+        def epok():
+            return time.time()
+
+        @staticmethod
+        def tid():
+            return time.ctime()
+
+        @staticmethod
+        def datum():
+            return datetime.date()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'year' in constructor call](https://app.codacy.com/manual/Enkelt/Enkelt-Dev/pullRequest?prid=5322890)